modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -4085,7 +4085,7 @@ AUI.add(
 							var defaultLocale = field.getDefaultLocale();
 
 							availableLanguageIds.forEach((locale) => {
-								if (!localizationMap[locale]) {
+								if (localizationMap[locale] === undefined) {
```

Comment:
@NemethNorbert do you think it is necessary to check that `localizationMap[locale] === null`?